R/estimateDiversity.R

```diff
@@ -199,15 +223,31 @@ setMethod("estimateRichness", signature = c(x = "SummarizedExperiment"),
     ans
 }
 
+.get_coverage <- function(x, threshold, ...){
```

Comment:
Set the default value for threshold here and remove it from the `setMethod` call. Should work without any other changes. Please describe the argument in the man page in any event

Reply:
![image](https://user-images.githubusercontent.com/60338854/110365958-58c45b80-804e-11eb-8d2e-21b583cdf112.png)

I'm wondering, why the `threshold` is not passed to .get_coverage. I think it should be because `x` is passed there. 

In `.run_diversity`, it is still 0.5 like it should be, but it is not passed to forward. In `.get_coverage` `threshold `is the default value, 0.9.

Reply:
Because the argument is stillnamed in the function definition of `.run_dvrsty`. If you remove it there, it will work.

If you `debug` this, you will noticed, that `list(...)` does not contain the threshold variable.

Reply:
You can have a look at a R specific explanation of the behavior of `...` here: https://adv-r.hadley.nz/functions.html?q=...#fun-dot-dot-dot

Reply:
Ok, thanks for the help. I think now it is working like it supposed to be

Reply:
One thing, that causes an error. How that `threshold` should be described, because now it is an additional parameter, and `@param threshold `is not possible?

Reply:
I usually describe it in the `@param ...` section using `\itemize{\item{\code{threshold}{... description ...}}`. There are probably some examples in `mia` or `miaViz`